asciidoc-markup-sample-doc/asciidoc-markup-samples.adoc

```diff
@@ -154,6 +159,13 @@ a|The `GET` operation can be used to do something.
 
 Answer by typing `Yes` or `No` when prompted.
 
+| Keyboard shortcuts
+a|
+....
+kbd:[key1(+key2+...)]
```

Comment:
Could you please include a specific example? Like `Ctrl+Alt+Del` or something?

Reply:
Yes.

Reply:
I added a specific example. Besides, I also added the links to Asciidoctor documentation and changed the document structure a little.